### PR TITLE
 Added maths and image upload with resize feature

### PR DESCRIPTION
**Pull request by jitendraashutec**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,153 @@
...
+
+## How to use 
+
+- Create dist file
+```
+    npm run package  // create dist
+
+```
+
+-- copy and past dist folder into your project root folder
```
Comment (milansar) [inline README.md]: correct paste spelling

File: README.md [hunk 2]
```diff
@@ -1,2 +1,153 @@
...
+
+```typescript
+
+// Math quill options
+ mathOptions = {
+    quill_options: {
+      maths: [{
+        cmd: 'sqrt', // Math quill 
+        name: 'Square Root',
+        url: '' // Image url of maths function
```
Comment (milansar) [inline README.md]: change comment to // image url of sqrt as button icon

File: README.md [hunk 3]
```diff
@@ -1,2 +1,153 @@
...
+      },
+      {
+        cmd: 'pm',
+        name: 'Plus Minus',
+        url: '' // Image url of maths function
```
Comment (milansar) [inline README.md]: same as above

File: angular.json [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+            "tsConfig": "projects/demo-app/tsconfig.app.json",
+            "assets": [
+              "projects/demo-app/src/favicon.ico",
+              "projects/demo-app/src/assets"
+            ],
+            "styles": [
+              "projects/demo-app/src/styles.css",
+              // "./dist/ng-quill-tex/assets/theme.css"
+            ],
+            "scripts": ["projects/demo-app/src/assets/js/nativescript-webview-interface.js"]
```
Comment (milansar) [inline angular.json:64]: add comment only required if using nativeScript integration

File: projects/ng-quill-tex/src/lib/ng-quill-tex.component.html [hunk 5]
```diff
@@ -0,0 +1,23 @@
+<div [formGroup]="group" *ngIf="group else noGroup">
+  <label> group </label>
+  <quill-editor [modules]="modules" #quillEditior trackChanges="all" format="object" (onEditorCreated)="getEditorInstance($event)"
+    [formControlName]="controlName">
+  </quill-editor>
+</div>
+
+<!-- <ng-template #formArrayName>
```
Comment (milansar) [inline projects/ng-quill-tex/src/lib/ng-quill-tex.component.html]: Remove this commented code

File: projects/ng-quill-tex/src/lib/quill-module/formula.ts [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+    return domNode.getAttribute('data-value');
+  }
+
+  constructor(node) {
+    super(node);
+    this.text = this.statics.value(node);
+  }
+
+  // length() {
+  //   return this.text.length;
```
Comment (milansar) [inline projects/ng-quill-tex/src/lib/quill-module/formula.ts]: remove commented code

**Comment by milansar** [comment]
@anihalaney , PR is ready for your review

File: .gitignore [hunk 7]
```diff
@@ -0,0 +1,4 @@
+node_modules
+dist
+.DS_Store
+package-lock.json
```
Comment (anihalaney) [inline .gitignore:4]: why are we ignoring package-lock.json?

**Pull request merged**